### PR TITLE
Address Book Query bug

### LINKED ISSUE #132: Query to get the latest address book fails
**Summary of the defect**
The New Query to get the latest address book has a bug. 
Add an alert in cloud watch to catch this error too.

Mainnet ERT got effected because of this as we had issues with GRPC connection to node 0.0.4 and we couldn't get the address book from the DB and thus disabling communication from ERT to Hedera network. 

**How to reproduce (if possible)**
All running ERT instances has this issue.

### PR DESCRIPTION
**Pull request by anighanta**
Signed-off-by: Anirudh Ghanta <anirudh.ghanta@hedera.com>

**Related issue(s)**:
Closes #132 

**Summary of the change**:
fixed the query to be single liner rather than formatting it to be a multiple line query

**External impacts**:
If applicable, list and explain how these change impact the tool's behaviour.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -185,6 +185,12 @@
             <version>1.61</version>
         </dependency>
 
+        <dependency>
+            <groupId>postgresql</groupId>
+            <artifactId>postgresql</artifactId>
+            <version>9.1-901-1.jdbc4</version>
```
Comment (cesaragv) [inline pom.xml:191]: This is an old version. You should use https://mvnrepository.com/artifact/org.postgresql/postgresql/42.2.18 instead. Unfortunately, this change requires you to validate everything again

File: pom.xml [hunk 2]
```diff
@@ -286,6 +292,19 @@
                     </execution>
                 </executions>
             </plugin>
+
+            <plugin>
```
Comment (cesaragv) [inline pom.xml:296]: Could you please confirm that you shouldn't be using this instead: https://mvnrepository.com/artifact/org.flywaydb/flyway-maven-plugin/7.3.2

File: pom.xml [hunk 3]
```diff
@@ -286,6 +292,19 @@
...
+                <groupId>com.googlecode.flyway</groupId>
+                <artifactId>flyway-maven-plugin</artifactId>
+                <version>1.7</version>
+                <configuration>
+                    <driver>org.postgresql.Driver</driver>
+                    <url>jdbc:postgresql://...</url>
+                    <user>...</user>
```
Comment (cesaragv) [inline pom.xml:303]: how does the `...` work?

File: src/main/java/com/hedera/exchange/database/ExchangeRateAWSRD.java [hunk 4]
```diff
@@ -79,27 +79,7 @@
...
-			"        (\n" +
-			"            SELECT\n" +
-			"                MAX(expirationTime) expirationTime \n" +
-			"            FROM\n" +
-			"                address_book \n" +
-			"            WHERE\n" +
-			"                address_book.networkName = ?\n" +
-			"        ) AS e2 \n" +
-			"            ON e1.e1_expirationtime = e2.expirationTime LIMIT 1";
+	private static final String LATEST_ADDRESS_BOOK_QUERY = "SELECT e1.expirationTime, e1.addressBook FROM address_book AS e1 INNER JOIN (SELECT MAX(expirationTime) expirationTime FROM address_book WHERE networkName = ?) AS e2 ON e1.expirationTime = e2.expirationTime LIMIT 1";
```
Comment (cesaragv) [inline src/main/java/com/hedera/exchange/database/ExchangeRateAWSRD.java:82]: I think it is preferable if the query can be read without scrolling right. Is there a particular reason this was moved to a one-liner?

**Review by cesaragv — CHANGES_REQUESTED** [review]

File: src/main/java/com/hedera/exchange/database/ExchangeRateAWSRD.java (hunk 4 above)
Comment (anighanta) [reply src/main/java/com/hedera/exchange/database/ExchangeRateAWSRD.java:82]: Getting formatting errors if I break it into separate lines. I tested the query which is split into multiple lines in dataGrip and got errors. Single line statement works best.

**Comment by anighanta** [comment]
closed in #142

**Pull request closed without merging**